src/chrome/content/rules/Hercrentals.com.xml

```diff
@@ -0,0 +1,48 @@
+<!--
+	Non-functional subdomains:
+		- airwatch	(expired cert)
+		- autodiscover	(connection refused)
+		- css		(SSL handshake failed)
+		- g		(redirects to invalid www.g)
```

Comment:
WFM

Reply:
it still redirects to `www.g` which refuse connection. connected via US location.

Reply:
It works for me in browser, but not with `curl`.
The DNS resolves to multiple endpoints.

Reply:
I see. I added a test url. should I redirect `www.g` (which is not resolvable) to `g`?

Reply:
Redirect doesn't help. Removed `g`.